### PR TITLE
f-template-subnav@v0.2.0 - Main layout of components and initial styling

### PR DESCRIPTION
**Pull request by oliversweb**
### Added
- f-breadcrumbs, f-navigation-links and styling

___

Example of the generic layout:

image

___

Example of it in use:

Desktop
image

Mobile
image

### CODE REVIEW

File: packages/components/organisms/f-template-subnav/src/components/TemplateSubNav.vue [hunk 1]
```diff
@@ -1,27 +1,102 @@
...
+}
+
+.c-navigation {
+    flex-basis: 15rem;
+    flex-grow: 1;
+}
+
+.c-navigation,
+.c-breadcrumb-container {
+    @include media('<=mid') {
```
Comment (dandel10n) [inline packages/components/organisms/f-template-subnav/src/components/TemplateSubNav.vue]: I checked that in prod nav is shown from 768px so I think this media query should not include mid
```suggestion
    @include media('<mid') {
```

File: packages/components/organisms/f-template-subnav/src/components/TemplateSubNav.vue [hunk 2]
```diff
@@ -1,27 +1,102 @@
...
-    components: { },
-    props: { }
+    name: 'TemplateSubNav',
+
+    components: {
+        BreadCrumbs,
+        NavigationLinks
+    },
+
+    props: {
```
Comment (dandel10n) [inline packages/components/organisms/f-template-subnav/src/components/TemplateSubNav.vue:40]: In the readme both props marked as required but they are not at the moment

**Review by simonsandersje — DISMISSED** [review]

File: packages/components/organisms/f-template-subnav/README.md [hunk 3]
```diff
@@ -67,12 +67,69 @@ export default {
...
 
-The props that can be defined are as follows (if any):
+| Prop  | Type  | Default | Required | Description |
+| ----- | ----- | ------- | ----- | ----------- |
+| breadcrumbs-links | array | [ ] | No (*but expected*) | An array of links objects (_see example #1 below_) |
+| navigation-links | array | [ ] | No (*but expected*) | An array of links objects (_see example #2 below_) |
 
-| Prop  | Type  | Default | Description |
-| ----- | ----- | ------- | ----------- |
+Example #1
```
Comment (minrwhite) [inline packages/components/organisms/f-template-subnav/README.md]: Should examples `#1` and `#2` be the other way around? `breadcrumbs-links` refers to `example #1`, but `selected` seems to be a property that would apply to the nav links?

Comment (oliversweb) [reply packages/components/organisms/f-template-subnav/README.md]: Yes good spot 👍

**Review by minrwhite — APPROVED** [review]

**Review by simonsandersje — APPROVED** [review]

**Pull request merged**